overlord/auth/auth.go

```diff
@@ -381,6 +382,16 @@ var (
 	ErrNoSerial = errors.New("no device serial yet")
 )
 
+// CloudInfo reflects cloud information for the system (as captured in the core configuration).
+type CloudInfo struct {
+	Name             string `json:"name"`
+	Region           string `json:"region,omitempty"`
+	AvailabilityZone string `json:"availability-zone,omitempty"`
+}
+
+// TODO: move AuthContext to something like a storecontext package, it
+// is about more than just authorization now.
+
 // An AuthContext exposes authorization data and handles its updates.
```

Comment:
Hm, this struct is doing a bit more nowdays, I wonder if we should update the name at some point. But not for this PR :)

Reply:
yes, if as per my TODO we move it to a dedicated package the name would not also not have Auth anymore in it